CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing to the Project
+
+This document contains and defines the rules that have to be followed by any
+contributor to the project, in order for any change to be merged into the
+stable branches.
+
+## Workflow
+
+The workflow followed is waterfall. In short, a contributor will target his 
+work on the lowest version branch possible starting with 2.0, and the changes
+will automatically be forward-ported to higher versions.
+
+## Restrictions of the master branch
+
+The master branch is a very specific branch in our workflow. No commit must
+ever go directly into the master branch, and the code shall follow one and only
+one path to get into the master branch.
+
+Merging a pull request into the master branch will only ever be done by the
+Bert-E merging bot. Once the necessary approvals and CI passes are met, one 
+only needs to comment `@Bert-E_scality approve` to trigger the merging process
+for one's pull request.
+
+## Workflow Guidelines
+
+### Branching Guidelines
+
+When branching, one should choose a name:
+
+* containing a reference to the GitHub issue being tackled
+* starting with a letter, due to a restriction in our merging bot
+* relevant to the goal of the changes made
+
+### Committing Guidelines
+
+No restrictions are placed at this time on individual commits passing in the
+CI and/or maintaining full functionality of the repository.
+
+Commit messages should:
+
+* have a short (< 60 characters) summary as title
```

Comment:
I believe the Git standard is 52

Reply:
Less than 50 in fact, according to the man page of `git commit`

> Though not required, it’s a good idea to begin the commit message with a single short (less than 50 character) line summarizing the change, followed by a blank line and then a more thorough description. The text up to the first blank line in a commit message is treated as the commit title, and that title is used throughout Git. For example, git-format-patch(1) turns a commit into email, and it uses the title on the Subject line and the rest of the commit in the body

The general rule is often called the "50/72 rule".

Reply:
Done